nominal/nominal.py

```diff
@@ -269,6 +269,10 @@ def create_run(
     start: datetime | str | ts.IntegralNanosecondsUTC,
     end: datetime | str | ts.IntegralNanosecondsUTC | None,
     description: str | None = None,
+    *,
+    properties: Mapping[str, str] | None = None,
+    labels: Sequence[str] = (),
+    attachments: Iterable[Attachment] | Iterable[str] = (),
```

Comment:
I didn't add the `asset` parameter even though it's defined on the client method, since there's been talks of potentially changing how runs are associated with assets